REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/SessionService.java

```diff
@@ -0,0 +1,23 @@
+package pl.jakubowskiprzemyslaw.REST.services;
+
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.Optional;
+
+@Service
+public class SessionService {
+
+  public void addObjectToSession(HttpServletRequest request, Object object) {
+    HttpSession session = request.getSession();
+    session.setAttribute(object.getClass().getSimpleName(), object);
+  }
+
+  public boolean isObjectInSession(HttpServletRequest request, String objectName) {
```

Comment:
you're sending request, but name of session suggests that it should accept request

Reply:
changed